acos_client/axapi_http.py

```diff
@@ -66,26 +64,42 @@ def merge_dicts(d1, d2):
 
 broken_replies = {
     ('<?xml version="1.0" encoding="utf-8" ?><response status="ok">'
-     '</response>'): '{"response": {"status": "OK"}}',
+     '</response>'): (json.dumps({"response": {"status": "OK"}})),
 
     ('<?xml version="1.0" encoding="utf-8" ?><response status="fail">'
      '<error code="999" msg=" Partition does not exist. '
      '(internal error: 520749062)" /></response>'):
-    ('{"response": {"status": "fail", "err": {"code": 999,'
-     '"msg": " Partition does not exist."}}}'),
+    (json.dumps({"response": {"status": "fail", "err": {"code": 999,
+     "msg": " Partition does not exist."}}})),
 
     ('<?xml version="1.0" encoding="utf-8" ?><response status="fail">'
      '<error code="999" msg=" Failed to get partition. (internal error: '
      '402718800)" /></response>'):
-    ('{"response": {"status": "fail", "err": {"code": 999,'
-     '"msg": " Partition does not exist."}}}'),
+    (json.dumps({"response": {"status": "fail", "err": {"code": 999,
+     "msg": " Partition does not exist."}}})),
 
     ('<?xml version="1.0" encoding="utf-8" ?><response status="fail">'
      '<error code="1076" msg="Invalid partition parameter." /></response>'):
-    ('{"response": {"status": "fail", "err": {"code": 1076,'
-     '"msg": "Invalid partition parameter."}}}'),
+    (json.dumps({"response": {"status": "fail", "err": {"code": 1076,
+     "msg": "Invalid partition parameter."}}})),
+
+    ('<?xml version="1.0" encoding="utf-8" ?><response status="fail">'
+    '<error code="999" msg=" No such aFleX. (internal error: '
+    '17039361)" /></response>'):
```

Comment:
To [PEP8](https://www.python.org/dev/peps/pep-0008/)-ify insert a single space before lines 87, 88, 90, 93,94, and 96. This will make it match the `(` as it is above (lines 66-84).